src/operations/crop.ts

```diff
@@ -0,0 +1,99 @@
+import { IJS } from '../IJS';
+import checkProcessable from '../utils/checkProcessable';
+
+export interface CropOptions {
+  /**
+   * Specify from what row the image should be cropped.
+   *
+   * @default 0
+   */
+  row?: number;
+  /**
+   * Specify from what colum the image should be cropped.
+   *
+   * @default 0
+   */
+  column?: number;
+  /**
+   * Specify the width of te cropped image
+   *
+   * @default image.width
+   */
+  width?: number;
+  /**
+   * Specify the width of te cropped image
+   *
+   * @default image.height
+   */
+  height?: number;
+}
+
+/**
+ * Crop the input image to a desired size.
+ *
+ * @param image - Image to crop
+ * @param [options] - Crop options.
+ * @returns The new cropped image
+ * @example
+ * var cropped = image.crop({
+ *   row:20,
+ *   column:100
+ * });
+ */
+export default function crop(image: IJS, options: CropOptions = {}) {
+  let {
+    row = 0,
+    column = 0,
+    width = image.width - column,
+    height = image.height - row,
+  } = options;
+
+  checkProcessable(image, 'crop', {
+    bitDepth: [8, 16],
+  });
+
+  row = Math.round(row);
+  column = Math.round(column);
+  width = Math.round(width);
+  height = Math.round(height);
+
```

Comment:
```suggestion
```

Let's remove the rounding. There's no obvious reason to support it.

Reply:
Done in 028e6cd0e5542c1b3025ca276e20e70686964cfe